doc/api/fs.md

```diff
@@ -293,8 +293,6 @@ A class representing a directory stream.
 
 Created by [`fs.opendir()`][], [`fs.opendirSync()`][], or [`fsPromises.opendir()`][].
 
-Example using async interation:
```

Comment:
Should the typo in this one be fixed too instead of removing the line?

Reply:
I think the line should be removed. In general, I dislike putting `for example:` and similar stuff before code that is obviously an example.